vms/avm/block/executor/block_test.go

```diff
@@ -46,6 +47,12 @@ func TestBlockVerify(t *testing.T) {
 				b := &Block{
 					Block: mockBlock,
 					manager: &manager{
+						backend: &executor.Backend{
+							Config: &config.Config{
+								DurangoTime: time.Time{},
+								EForkTime:   mockable.MaxTime,
+							},
```

Comment:
E fork is likely to introduce changes to the fee mechanism in X and P-chains.
We update UTs here to make sure they won't break following the new fee mechanism introduction.
We'll update them to work E-fork on a test by test base later on